pkgs/development/compilers/unison/default.nix

```
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "unison-code-manager";
+  milestone_id = "M1c";
+  version = "1.0.${milestone_id}-alpha";
+
+  src = if (stdenv.isDarwin) then
+    fetchurl {
+      url = "https://github.com/unisonweb/unison/releases/download/release/${milestone_id}/unison-osx.tar.gz";
+      sha256 = "03q02r7qc7ybqz16kmpk2d8l9vx28kaj9x59mlxzi8a4mr0j3vzb";
+    }
+  else
+    fetchurl {
+      url = "https://github.com/unisonweb/unison/releases/download/release/${milestone_id}/unison-linux64.tar.gz";
+      sha256 = "1iwynqnp1i39pyq9wc01x7y22y1qa0rrjlx40jjdgnj23y1r6jk4";
+    };
+
+  # The tarball is just the prebuilt binary, in the archive root.
+  sourceRoot = ".";
+  phases = "unpackPhase installPhase fixupPhase";
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv ucm $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Modern, statically-typed, purely functional, next-generation programming language, currently in development";
+    homepage = http://unisonweb.org/posts/;
+
+    license = licenses.bsd3;
+    maintainers = maintainers.virusdave;
```

```suggestion
    maintainers = [ maintainers.virusdave ];
```

fixes eval failure, need to be a list

---

Done!